Fix PNG seek

hugovk opened the pull request:
Fixes #4509.

Changes proposed in this pull request:

 * Initialise `__frame` to 0 to fix `seek` after open

pmrowla commented on src/PIL/PngImagePlugin.py:
```diff
@@ -630,6 +630,7 @@ class PngImageFile(ImageFile.ImageFile):
 
     format = "PNG"
     format_description = "Portable network graphics"
+    __frame = 0
```
really minor, but initializing this in `_open()` would make it more consistent w/the other existing image plugins (I checked  FLI/MPO/TIFF)

hugovk commented:
Thanks, updated!

Also added a test for `tell`.

(Travis CI has a [disk space error](https://travis-ci.org/github/python-pillow/Pillow/jobs/670002050) on the s390x job and no restart button, but it's all green on my [fork](https://travis-ci.org/github/hugovk/Pillow/builds/670002027).)

almarklein commented:
Thanks for the quick fix 👍
